pyproject.toml

```diff
@@ -69,8 +69,7 @@ doc = [
   "jupyterlite-sphinx",
   "sphinxcontrib-youtube<1.4",
   "sphinx-favicon>=1.0.1",
-  # Install nbsphinx in case we want to test it locally even though we can't load
-  # it at the same time as MyST-NB.
+  "ipykernel",
```

Comment:
Similar question here: was `ipykernel` previously included as a dependency of `myst-parser`? And it doesn't come with `nbsphinx`? So it's a peer dependency of `nbsphinx` or something like that?

Reply:
yep nbsphinx took the same path as jupyter-sphinx of not setting ipykernel as a direct deps. but they are slowly moving toward it (it was dropped by another of their dependencies.